packages/react-pdf-toolkit/src/utils/is-link.ts

```diff
@@ -0,0 +1,9 @@
+export const isLink = (value: string): boolean => {
```

Comment:
Redundant return type.
```suggestion
export const isLink = (value: string) => {
```

Reply:
P3